### ISSUE TITLE
Please update the reference tfp.bijector.BatchNormalization

### ISSUE THREAD

**Issue by MokkeMeguru** [tensorflow/probability#587]
Here is the colab about the Problem.

https://colab.research.google.com/gist/MokkeMeguru/055440bb1eefbe58badbfe81eab73409/batchnormalization_test.ipynb

- The main problem is your reference uses some old functions. Please update them.

- Another problem is `event_shape` for BatchNormalization.    
BatchNormalization is affected *over* the batch axis.     
But `event_shape ` in `tfdTransformedDistribution` usually demands the shapes without the batch axis.     
I'm wondering if this transformation works correctly.

Thanks!
